File: js/pigatize.js
var vowels = ["a", "e", "i", "o", "u"]

var pigatize = function(word) {
  for (var i = 0; i < vowels.length; i++) {
    if (word[0] === vowels[i]) {
      var newWord = word + "way";
      return console.log(newWord);
    }
  }
  var newWord = word.slice(1) + word[0] + "ay";
  console.log(newWord);
}

pigatize("banana");
pigatize("open");
pigatize("apple");

File: rb/pigatize.rb

def pigatize(word)
  vowels = ["a", "e", "i", "o", "u"]
  if vowels.include?(word[0])
    new_word = word + "way"
    puts new_word
  else
    new_word = word.slice(1..-1) + word[0] + "ay"
    puts new_word
  end
end

pigatize("banana")
pigatize("open")
pigatize("apple")
